1.7/administration/securing-your-cluster.md

```diff
@@ -29,6 +29,41 @@ the other nodes in the cluster via URL routing. For security, the DC/OS cloud
 template allows configuring a whitelist so that only specific IP address
 ranges are permitted to access the admin zone.
 
+#### Steps for Securing Admin Router
+
+By default, Admin Router will permit unencrypted HTTP traffic. This is not
+considered secure, and you must provide a valid SSL certificate and redirect
+all HTTP traffic to HTTPS in order to properly secure access to your cluster.
+
+Once you have a valid SSL certificate, install the certificate on each master.
```

Comment:
I wonder if it might be worth expanding a little bit here on how to install the cert on the masters? It seems that they can put it anywhere and just provide the path as noted below. What about the file permissions in that case and how should they protect the private key? Also, should the install the same cert on each master or different certs? These are some of the questions that come to my mind, at least.